torch/_torch_docs.py

```diff
@@ -1780,6 +1785,11 @@ def merge_dicts(*dicts):
 Divides each element of the input ``input`` with the scalar ``other`` and
 returns a new resulting tensor.
 
+.. warning::
+    Integer division using div is deprecated and will be removed in a future
```

Comment:
as above, be explicit with the version numbers.

Reply:
Sounds good.